### DISCUSSION TITLE
Traducción de «spread syntax»

### DISCUSSION THREAD

**Discussion by carburo** [reactjs/es.reactjs.org#600] [Q&A] [answered]
En la documentación anterior se usó a menudo «sintaxis de propagación», pero nunca se incluyó como recomendación en el glosario. Me parece mejor que las alternativas que usa la traducción de MDN:

- Sintaxis _spread_ (conserva el anglicismo crudo, aún cuando que es un término que se debería traducir).
- Sintaxis extendida (una mala traducción, porque lo que se extiende —o mejor, propaga— no es la sintaxis).

Creo que es preferible «propagación» a «extensión» porque la segunda puede dar a entender que se está aumentando algo (que no es el caso) o también puede confundirse con la terminología de herencia de la programación orientados a objetos.

¿Existe otra alternativa a considerar antes de añadir una recomendación en el glosario?

**Comment by erikgiovani** [comment]
Suena razonable, yo lo conozco por propagación, también al spread operator se le traduce como operador de propagación

**Comment by carburo** [comment, marked as answer]
Ya fue añadida al glosario «sintaxis de propagación».
